content/en/docs/concepts/cluster-administration/system-traces.md

```diff
@@ -76,6 +76,36 @@ samplingRatePerMillion: 100
 For more information about the `TracingConfiguration` struct, see
 [API server config API (v1alpha1)](/docs/reference/config-api/apiserver-config.v1alpha1/#apiserver-k8s-io-v1alpha1-TracingConfiguration).
 
+### Trace instrumentation for the kubelet {#kubelet-traces}
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+The kubelet CRI interface and authenticated http servers are instrumented to generate
+trace spans. As with the apiserver, the endpoint and sampling rate are configurable.
+Trace context propagation is also configured. When enabled without a tracing configuration sampling rate,
```

Comment:
For the sampling rate, the parent span's sampling decision is always respected first (i.e. always sample if parent is sampled, never sample if parent is not sampled).  The configured sampling rate applies to spans without a parent.

Reply:
updated, it now reads:
```
A parent span's sampling decision is always respected.
A provided tracing configuration sampling rate will apply to spans without a parent.
Enabled without a configured endpoint, the default OpenTelemetry Collector reciever address of "localhost:4317" is set.
```

Reply:
That works for me!